Automate/Speckle.Automate.Sdk/Runner.cs

```diff
@@ -85,7 +86,7 @@ public static async Task<int> Main(string[] args, Func<AutomationContext, Task>
   }
 
   /// <summary>
-  /// Main entrypoint to execute an Automate function with input data of type <see cref="TInput"/>
+  /// Main entrypoint to execute an Automate function with input data of type.
```

Comment:
Why was this deleted? cref references to types are usually better as they allow to navigate to them on the xml docs

Reply:
The warning was pushing me I believe 🤔

Reply:
I think the correct tag here would be `<TypeParamRef cref="TInput/>`

Reply:
It is already there at line 93

Reply:
no that's the `typeparam` tag. this would be a reference to the name of that type param
If you look at the rendered doc panel, you'll see the indented summary tag would read "...input data of type TInput"

![image](https://github.com/specklesystems/speckle-sharp/assets/45512892/2dbdb6bd-c413-4e3c-b6ad-0e0035a49e17)

Reply:
I can walk you through some of the XML doc tags after standup if you'd like, there's several lesser know ones like `typeparamref` that come in handy when writing XML docs

Reply:
gotcha, yup there are different. I'm adding it as in your screenshot, then we can chat about it after SU